src/__experimental__/components/checkbox/__snapshots__/checkbox.spec.js.snap

```diff
@@ -436,7 +600,6 @@ exports[`Checkbox base theme renders as expected 1`] = `
             aria-labelledby="guid-12345-label"
             className="c8 c9"
             id="guid-12345"
-            name="my-checkbox"
```

Comment:
`name` prop is not passed to input anymore

Reply:
Changed, thanks